qiskit/aqua/components/optimizers/aqgd.py

```diff
@@ -39,150 +41,307 @@ class AQGD(Optimizer):
 
     Gradients are computed "analytically" using the quantum circuit when evaluating
     the objective function.
-    """
 
-    _OPTIONS = ['maxiter', 'eta', 'tol', 'disp']
+    """
+    _OPTIONS = ['maxiter', 'eta', 'momentum', 'ptol', 'otol', 'averaging']
 
     def __init__(self,
-                 maxiter: int = 1000,
-                 eta: float = 3.0,
-                 tol: float = 1e-6,
-                 disp: bool = False,
-                 momentum: float = 0.25) -> None:
+                 maxiter: Optional[List[int]] = None,
+                 eta: Optional[List[float]] = None,
+                 momentum: Optional[List[float]] = None,
+                 param_tol: float = 1e-6,
+                 objective_tol: float = 1e-6,
+                 averaging: int = 10) -> None:
         """
+        Constructor.
+
```

Comment:
Can we delete this. It is self-evident and this ends up in the html which does then not look so great.

Reply:
Done.